assignment2.txt

```diff
@@ -0,0 +1,11 @@
+Bettina Benitez
+
+1. You can combine two or more operators. However, you must include && or ||.
+2. Logical errors happen when the output of the code is unexpected while Syntax errors are grammatical errors in the code.
+3. The conditional statement is true
+4. No because there are too many statements at once. It should be separated by &&
+5.
+i. It is invalid because you do not use "then" because it is not part of the c language. It also needs curly brackets before and after "c = 0"
```

Comment:
You're right about the then, but since there's only one line for the if statement, it doesn't necessarily NEED the curly braces